packages/formstr-app/src/components/SafeMarkdown/ColorfulMarkdownInput.tsx

```diff
@@ -128,9 +129,13 @@ export const ColorfulMarkdownTextarea: React.FC<Props> = ({
         </div>
       ) : (
         <Input.TextArea
-          value={value}
+          value={getPlainText()}
```

Comment:
Instead of of replacing span tags can we just track them in different variables, this will cause problems in cases where the user actually wants to write a span tag in the question